scripts/o3de/ExportScripts/export_source_android.py

```diff
@@ -123,6 +125,9 @@ def export_source_android_project(ctx: exp.O3DEScriptExportContext,
                 logger.error("Unable to verify signing configuration in build.gradle. Aborting deployment...")
         except FileNotFoundError:
             logger.error("Unable to open build.gradle file. Aborting deployment...")
+    
+    if open_output_dir:
+        webbrowser.open(target_android_project_path)
```

Comment:
I'd still rather not have the script open the containing folder, i'd rather not provide it at all for this iteration than to hack in an additional argument to do so.

Reply:
We should just write the location of the output folder to the log as the final line instead. The Project Manager user should have a 'show log' button anyways

Reply:
so should I go ahead and remove it, and make a follow-up GHI for that functionality as a request instead?

Reply:
and I can add it as a log statement, certainly

Reply:
yes, lets remove it. It potentially can write the new config settings that you defined locally, which we don't want